common-util/src/test/java/org/sunbird/services/sso/impl/KeyCloakRsaKeyFetcherTest.java

```diff
@@ -0,0 +1,39 @@
+package org.sunbird.services.sso.impl;
+
+import java.security.PublicKey;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.sunbird.common.models.util.KeyCloakConnectionProvider;
+
+/**
+ * @author github.com/iostream04 This class will Test
+ *         KeyCloakRsaKeyFetcherTest's getPublicKeyFromKeyCloak method .
+ */
+
+public class KeyCloakRsaKeyFetcherTest {
+  public static final String FAIL_TEST = "fail-check";
+
+  @Test
+  public void getPublicKeyFromKeyCloakTest() {
```

Comment:
I will merge the PR. However, raise another PR to properly mock the key cloak API endpoint. Check with @manzarul in case of any doubt.